db/models/interview.js

```diff
@@ -139,7 +139,114 @@ InterviewerCalendlyLinks.init(
   { sequelize, modelName: "InterviewerCalendlyLinks" }
 );
 
+class InterviewQuestions extends Model {}
 
+InterviewQuestions.init(
+	{
+		id: {
+			type: DataTypes.INTEGER,
+			primaryKey: true,
+			autoIncrement: true,
+			allowNull: false,
+		},
+		InterviewRoundId: {
+			type: DataTypes.INTEGER,
+			allowNull: false,
+		},
+		question: {
+			type: DataTypes.STRING,
+			allowNull: false,
+		},
+		questionType: {
```

Comment:
Let's not reuse the word "question" in attribute name if it is already in the model name.
Because then our code will contain repetitions like this:
```javascript
const question = InterviewQuestion.get()
console.log(question.questionType)
```

If named `type`, it will be:
```javascript
const question = InterviewQuestion.get()
console.log(question.type)
```